index.js

```diff
@@ -1,15 +1,25 @@
-var /**
-     * @private
-     * @type {Object}
-     */
-    db = require('./db.json');
+/**
+ * @typedef {Object} HTTPCode
+ * @property {String} name HTTP Reason Phrase
+ * @property {String} detail Description of the HTTP Status code
+ */
+
+/**
+ * @module http-reasons
+ */
+
+var db = require('./db');
 
+/**
+ * Exposes methods to access HTTP Code DB
+ * @type {Object}
+ */
 module.exports = {
     /**
-     * @param {String|Number} code
-     * @returns {Object} - {name:String, detail:String}
+     * @param {String} code HTTP Status Code
```

Comment:
It takes number too since it is cast to string. If we don't document like that, users will needlessly cast numbers to string before sending here. Also status code returned by system is a number.